build-tools/xaprepare/xaprepare/Application/HomebrewProgram.MacOS.cs

```diff
@@ -190,6 +190,11 @@ protected override async Task AfterDetect (bool installed)
 				return;
 			}
 
+			// It may happen that the package is installed but not linked to the prefix that's in the user's PATH.
+			// Detecting whether the package is linked would require requesting and parsing JSON for all the packages which
+			// would be more trouble than it's worth. Let's just link the package
+			await runner.Link (Name, echoOutput: false, echoError: false);
```

Comment:
Why *wouldn't* we want to echo stdout & stderr?

Reply:
See my reply above. The `Warning` line is printed to `stderr` and would be shown in red, the remaining two lines are sent to `stdout`.